spec/controllers/bulk_intervention_assignments_controller_spec.rb

```diff
@@ -68,7 +74,9 @@ def make_request(params = {})
           end
         end
         context 'missing intervention type' do
-          let(:params) { { bulk_intervention_assignment: { student_ids: [student.id] } } }
+          let(:params) do
+            { bulk_intervention_assignment: create_valid_bulk_assignment_params.except(:intervention_type_id) }
```

Comment:
This started failing with the stricter intervention validation, so this fixes it up.